ddl/src/main/resources/templates/kinetica/table/create.sqlt

```diff
@@ -1 +1,6 @@
-CREATE TABLE "[(${schemaName})]"."[(${tableName})]"([(${tableCode})]);
\ No newline at end of file
+[# th:if="${tableType} == null or ${tableType} == ''"]
+CREATE TABLE "[(${schemaName})]"."[(${tableName})]"([(${tableCode})]);
+[/]
+[# th:if="${tableType} != null and ${tableType} != ''"]
+CREATE [(${tableType})] TABLE "[(${schemaName})]"."[(${tableName})]"([(${tableCode})]);
```

Comment:
Based on the method `extractTableType` can we say that this `.sqlt` can work having only this line:

```
CREATE [(${tableType})] TABLE "[(${schemaName})]"."[(${tableName})]"([(${tableCode})]);
```

The `[(${tableType})] ` can either have value/s or be empty.

Reply:
I have also modified this part, now we will not have the need to check whether the value is empty or not because I have also included the 'TABLE' part inside the tableType variable.